### PR TITLE
How to Install Zabbix on CentOS 7

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

File: tutorials/install-zabbix-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+Password updated successfully!
+Reloading privilege tables..
+ ... Success!
+```
+
+For the rest of the questions, you should simply hit the "ENTER" key through each prompt to accept the default values. This will remove some sample users and databases, disable remote root logins, and load these new rules so that MySQL immediately respects the changes we have made.
+
+## Step 3 - Install Zabbix Repository
+
+Zabbix isn't available in our package manager by default, so we will install a repository configuration package using the official Zabbix repository package on [Zabbix Official Website](https://www.zabbix.com/download) for CentOS:
```
Comment (n-se) [inline tutorials/install-zabbix-on-centos-7/01.en.md]: Is there no repository or something similar? You should add a way on how to keep track of new updates. Zabbix itself does not check for new updates, does it?

File: tutorials/install-zabbix-on-centos-7/01.en.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+```
+
+In DBPassword use Zabbix database password for MySQL.
+
+## Step 8 - Starting Zabbix server process
+
+It's time to start Zabbix server process:
+
+```bash
+systemctl start zabbix-server
```
Comment (n-se) [inline tutorials/install-zabbix-on-centos-7/01.en.md]: You should also `systemctl enable zabbix-server` here

File: tutorials/install-zabbix-on-centos-7/01.en.md (hunk 1 above)
Comment (1817666) [reply tutorials/install-zabbix-on-centos-7/01.en.md]: I think the best methode for installing Zabbix monitoring is installing from packages.
https://www.zabbix.com/documentation/4.0/manual/installation/install_from_packages/rhel_centos

I will add another tutorial like "Configure Zabbix Monitoring to Send Alert (Email, Push, ...)" and "Configure Zabbix Agent (Linux, Windows)" and ... and I explain how can update zabbix, import and export database and ...

File: tutorials/install-zabbix-on-centos-7/01.en.md (hunk 2 above)
Comment (1817666) [reply tutorials/install-zabbix-on-centos-7/01.en.md]: Edited.

File: tutorials/install-zabbix-on-centos-7/01.en.md (hunk 1 above)
Comment (n-se) [reply tutorials/install-zabbix-on-centos-7/01.en.md]: I'm looking forward to the sequel, perfect!

**Comment by 1817666** [comment]
Hi @leojakeson 

Please edit line 193.

Edited:
`## Step 9 - Complete Zabbix Web Installer Wizzard`
To:
`## Step 9 - Complete Zabbix Web Installer Wizard`

Thanks :)

**Pull request merged**